src/Mod/Arch/ArchFloor.py

```diff
@@ -47,11 +47,32 @@ def QT_TRANSLATE_NOOP(ctxt,txt):
 __author__ = "Yorik van Havre"
 __url__ = "http://www.freecadweb.org"
 
+__doc__ = """This module provides tools to build Floor objects. Floors are used
```

Comment:
I personally suggest you to add the module docstring at the top of the file, not inside the `__doc__` variable. The docstring should come immediately after the LGPL2 license block.

Automatic documentation builders like Sphinx should read the topmost documentation string and display it for you. I haven't checked if Sphinx reads the `__doc__` variable properly; maybe it does, but placing the documentation string at the top of the file is more common than adding it to the variable.

Reply:
Resolved with ed6c261518cd6ee3e5c23e240e91e093a887ca46